dashboard/app/helpers/application_helper.rb

```diff
@@ -63,14 +63,11 @@ def activity_css_class(result)
   end
 
   def level_info(user, script_level, user_levels)
-    result = nil
     if user
-      ul = user_levels[script_level.level_id]
-      result = ul.try(:best_result) if ul
+      user_levels[script_level.level_id].try(:best_result) || 0
     else
-      result = client_state.level_progress(script_level)
+      0
```

Comment:
Alternatively, if we're certain this is only used in one place, move it to users_helper and mark it private and then you can do anything you want to the signature. (However, given that Ruby is dynamic and not type checked, it's a little difficult to prove that fact!)